charts/rmd/templates/configmap.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "rmd.fullname" . }}
+  labels:
+    {{- include "rmd.labels" . | nindent 4 }}
+    app.kubernetes.io/component: backend
```

Comment:
This is inconsistent with the rest of the chart that uses "rmd". I don't know the rmd architecture. Does rmd really need the component label? Usually the label is meant for applications that are built from several components (think microservices).